src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -184,26 +186,69 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
     datumHash,
   })
 
+  const prepareDatumHash = (
+    output: TxTypes.TransactionOutput,
+  ) => {
```

Comment:
No return value type given, also in prepareInlineDatum.

Reply:
Fixed in https://github.com/vacuumlabs/cardano-hw-cli/pull/127/commits/5e4905e60638b8421a1867207a17530056de6734.